src/report.tex

```diff
@@ -492,7 +496,8 @@ \subsubsection*{Vacuum Vessel}
 plasma, port components were used from the file CLITE\_V1\_REV131031\_SPACECLAIM 
 that was received from ITER. The vacuum vessel geometry was modified in several 
 ways. The simple components of the BL-Lite geometry were modified to include 
-the in-wall plates within the vacuum vessel shells. Also, overlaps were removed 
+the in-wall plates, with all detail retained. within the vacuum vessel shells.
```

Comment:
What does "all detail retained" mean?  

And this should be "retained," not "retained."  (comma)